src/components/VaultLocationReport.tsx

```diff
@@ -172,7 +172,12 @@ export default function VaultLocationReport(props: Props): ReactElement {
                       label='Muster remarks'
                     />
                   </ItemsReport>
-                  <ReportSignature id={id} />
+                  <ReportSignature>
+                    <Count
+                      resource={constants.R_ITEMS}
+                      filter={{ vaultLocation: id, loanedTo: undefined }}
```

Comment:
Aah, well spotted.  But, for SQLite to work, I think `loanedTo` needs to be `null`.  We also need to filter for `dispatchedDate: null` and `destroyedDate: null`.

_Hopefully_ we're using `null` across the Mock and SQLite databases now.

Reply:
The `filter` is just being copied from where the `items` are being listed. Changing the filter in the `Count` may result in different number then actually items listed. So changing here means changing above also

![image](https://github.com/DeepBlueCLtd/RCO/assets/50602678/0dc9ac0a-e0a4-4722-87cc-d5fae45f11fd)

Reply:
Yes - that's well spotted, thanks.